anitya/lib/backends/maven.py

```diff
@@ -70,3 +74,50 @@ def get_versions(cls, project):
                           .replace(':', '/'))
 
         return get_versions_by_regex(url, REGEX, project)
+
+    @classmethod
+    def check_feed(cls):
+        '''check_feed for Maven backend.
+
+        Return a generator over the latest 40 packages to Maven central index by
+        calling maven-index-checker application
+
+        Returns:
+            generator over new packages
+        '''
+        maven_url = 'http://repo2.maven.org/maven2'
+        if APP.config['JAVA_PATH'] is None:
+            raise AnityaPluginException('no java binary specified')
+        if APP.config['JAR_NAME'] is None:
+            raise AnityaPluginException('no maven-release-checker jar file specified')
+
+        try:
+            data = check_output([APP.config['JAVA_PATH'], "-jar",
+                                 APP.config['JAR_NAME']], universal_newlines=True)
+        except CalledProcessError:
+            raise AnityaPluginException('maven-release-checker exited with non zero value')
+
+        data = json.loads(data)
+        session = init(APP.config['DB_URL'])
```

Comment:
Nothing else jumped out at me, so waiting for now is good. I _will_ get to it this week, I promise :smile:. Stuff came up last week that I still haven't quite finished dealing with. Sorry about that!